### DISCUSSION TITLE
Does the wasm version works ?

### DISCUSSION THREAD

**Discussion by TeddyAlbina** [101arrowz/fflate#167] [General]
Hello,

I check the repo code and I've seen that a wasm version exists. Does it work ? 

I'm asking that question because we have problem when using fflate inside a worker on safari, because this browser lack support for nested workers.

Best regards.

**Comment by 101arrowz** [comment]
Unfortunately I haven't been able to make the WASM version work very well; it's generally much slower than the JS version. If you are inside a web worker in Safari, you unfortunately only have the option of using the synchronous APIs, but that should be OK since it won't block the main thread. You'll miss out on some performance gains from parallelization though.
